test/rails/action_controller/test_assertions.rb

```diff
@@ -21,7 +21,7 @@ def test_assert_template
   def test_routing_assertions
     params = { :controller => "models", :action => "index" }
     path = "/models"
-    assert_generates path, params
+    assert_generates path, params.clone
```

Comment:
This might be remove if [this patch](https://github.com/rails/rails/pull/23396) gets merged.